README.md

```diff
@@ -38,7 +38,7 @@ If you find this template does not fit your needs, we would appreciate if you co
 **Model source:** {{ [YeastMetabolicNetwork](http://doi.org/10.1038/nbt1492) }}  
 **Omic source:** {{ genomics; metabolomics }}  
 **Taxonomic name:** {{ _Saccharomyces cerevisiae_ }}  
-**Genome ID:** {{ provide identifier from PATRIC or GenBank }}  
+**Genome ID:** {{ provide identifier from PATRIC，GenBank, or other sources }}  
```

Comment:
I think it's a good practice for a standard to recommend only the best solutions. Therefore I would avoid writing "other sources" - if they are good they should be named, otherwise their use should not be encouraged. Anyway, people who want to use other sources can still do so.

Reply:
Yes, genome ids from GenBank or RefSeq would be ideal and should be encouraged. But there are other sources (e.g. Ensembl) with added annotation, or even self-made annotation. The `standard-GEM` should also be open for exceptions.

Reply:
In the case of _S.cerevisiae_, [Ensembl](https://fungi.ensembl.org/Saccharomyces_cerevisiae/Info/Index) provides a ENA identifier for the genome assembly. Via _identifiers.org_, that same id is available through [`insdc.gca:GCA_000146045.2`](https://identifiers.org/insdc.gca:GCA_000146045.2) on GenBank. So far it looks like recommending GenBank is sufficient.

Nevertheless, at the beginning of the Readme it is stated
> This is the `README.md` template provided by [standard-GEM](https://github.com/MetabolicAtlas/standard-GEM) and was crafted to cover most use-cases.

> If you find this template does not fit your needs, we would appreciate if you could report this by creating a new issue on [standard-GEM](https://github.com/MetabolicAtlas/standard-GEM/issues).


I believe these lines make it clear that the _entire_ Readme is flexible.

Reply:
Could you please update your PR and mention "other sources such as PATRIC or KBase" at the end of the [explanation about genome identifier](https://github.com/MetabolicAtlas/standard-GEM/blame/feat/genome-id/README.md#L33)?